src/util/navigation.js

```diff
@@ -8,7 +8,8 @@ import { host } from 'config'
 
 export const HYLO_ID_MATCH = '\\d+'
 export const POST_ID_MATCH = HYLO_ID_MATCH
-export const DEFAULT_POST_TYPE_CONTEXT = 'p'
+export const SLUG_MATCH = '.+'
+export const DEFAULT_POST_TYPE_CONTEXT = 'stream'
```

Comment:
stream is not a post type context, its a view like map. maybe we just need to call all these things views, and get rid of the idea of POST_TYPE_CONTEXT